minicluster/src/main/java/org/apache/accumulo/miniclusterImpl/MiniAccumuloClusterControl.java

```diff
@@ -111,6 +127,53 @@ public void adminStopAll() throws IOException {
     }
   }
 
+  private static TExternalCompactionList getRunningCompactions(ClientContext context)
+      throws TException {
+    Optional<HostAndPort> coordinatorHost =
+        ExternalCompactionUtil.findCompactionCoordinator(context);
+    if (coordinatorHost.isEmpty()) {
+      throw new TTransportException("Unable to get CompactionCoordinator address from ZooKeeper");
+    }
+    CompactionCoordinatorService.Client client = ThriftUtil.getClient(
+        new CompactionCoordinatorService.Client.Factory(), coordinatorHost.get(), context);
+    try {
+      TExternalCompactionList running =
+          client.getRunningCompactions(TraceUtil.traceInfo(), context.rpcCreds());
+      return running;
```

Comment:
Some of this could be tightened up for readability:
```suggestion
    var client = ThriftUtil.getClient(new CompactionCoordinatorService.Client.Factory(), coordinatorHost.get(), context);
    try {
      return client.getRunningCompactions(TraceUtil.traceInfo(), context.rpcCreds());
```

Reply:
I tightened this up a bit in [851c2c3]